maven-scm-api/src/main/java/org/apache/maven/scm/CommandParameters.java

```diff
@@ -53,11 +53,6 @@ public String getString( CommandParameter parameter )
     {
         Object object = getObject( String.class, parameter );
 
-        if ( object == null )
-        {
-            throw new ScmException( "Missing parameter: '" + parameter.getName() + "'." );
-        }
-
```

Comment:
Well, this will change to NPE. Please explain.

Reply:
You are right:
https://github.com/apache/maven-scm/blob/97ef7104b6847502d72469ae816b2720d879c45f/maven-scm-api/src/main/java/org/apache/maven/scm/CommandParameters.java#L327-L338

I don't understand its reasoning. A getter should get the value regardless null or not. Lets leave it as-is.